command/commands/team.py

```diff
@@ -110,7 +102,11 @@ def get_desc(self):
 
     def get_help(self):
         """Return command options for team events."""
-        return self.help
+        res = "\n*" + self.command_name + " commands:*```"
```

Comment:
Would it be possible to capitalize the command names? I think that would look a bit more professional.

Reply:
yep